features/step_definitions/web_steps.rb

```diff
@@ -31,7 +31,7 @@ def with_scope(locator)
 
 When /^(?:|I )press "([^"]*)"(?: within "([^"]*)")?$/ do |button, selector|
   with_scope(selector) do
-    click_button(button)
+    find_button(button).trigger('click')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.